cli/expiring.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"database/sql"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/cloudflare/cfssl_trust/common"
+	"github.com/cloudflare/cfssl_trust/model/certdb"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var expiringCmd = &cobra.Command{
+	Use:   "expiring",
```

Comment:
It shows certificates that *will* expire (and coincidentally, certificates that have expired), so it seems better to use this terminology.